scripts/test_ci_native_targets.py

```
@@ -0,0 +1,216 @@
+    ]
+
+
+def expected_proof_lines(shell: str, runner_arch: str, go_target: str) -> list[str]:
+    """Return the exact straight-line proof required for one native lane."""
+    if shell == "pwsh":
+        return [
+            'Write-Output "runner architecture: $env:RUNNER_ARCH"',
+            '$goTarget = "$(go env GOOS)/$(go env GOARCH)"',
+            'Write-Output "Go target: $goTarget"',
+            f'if ($env:RUNNER_ARCH -ne "{runner_arch}") '
+            f'{{ throw "expected {runner_arch} runner, got $env:RUNNER_ARCH" }}',
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)